packages/lib/server/revalidateCalendarCache.ts

```diff
@@ -1,18 +0,0 @@
-import type { NextApiResponse } from "next";
-
-import dayjs from "@calcom/dayjs";
-
-export const revalidateCalendarCache = (
```

Comment:
@roae @emrysal @zomars Let me know if you all would prefer to leave this as an empty implementation that we can come back to and replace with something like Redis cache. I don't think we should leave dead code around and we can always reference this commit when rethinking caching but I'm open.

Reply:
I prefer to remove it, we should avoid dead code.